ra_aid/agent_utils.py

```
@@ -56,58 +61,189 @@
+        provider_tokens = models_tokens.get(provider, {})
+        token_limit = provider_tokens.get(model_name, None)
+        if token_limit:
+            logger.debug(
+                f"Found token limit for {provider}/{model_name}: {token_limit}"
+            )
+        else:
+            logger.debug(f"Could not find token limit for {provider}/{model_name}")
+
+        return token_limit
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)